packages/oui-action-menu/src/action-menu.html

```diff
@@ -1,4 +1,4 @@
-<oui-dropdown align="{{::$ctrl.align}}"
+<oui-dropdown data-align="{{::$ctrl.align}}"
```

Comment:
Why add `data-` only on `align` attribute and not also on `arrow` on this case ? Is it the root cause of the problem you were fixing ?

Reply:
"Fixed" with a comment.